index.js

```diff
@@ -0,0 +1,15 @@
+const hamburger = document.querySelector('.hamburger');
```

Comment:
Almost there but please stick to the given Figma template for the design. Add some border radius and give some space as I marked below.
<img width="326" alt="Screenshot 2023-01-03 at 9 52 09 PM" src="https://user-images.githubusercontent.com/25887253/210424383-694f798c-082a-48c8-bd45-1cb99a668935.png">
<img width="385" alt="Screenshot 2023-01-03 at 9 54 43 PM" src="https://user-images.githubusercontent.com/25887253/210424384-55f16d35-f366-490e-b525-e586c135d38f.png">